### DISCUSSION TITLE
“How to Give a Talk” talk

### DISCUSSION THREAD

**Discussion by cassidyjames** [elementary/edw#9] [Withdrawn or Not Accepted]
Speaker: @danrabbit

Giving a talk—especially for a new online-only event like edw—can be daunting. This talk should debut ahead of time as a demonstration of the format, plus a practical guide for preparing and giving your own edw talk.

**Comment by danirabbit** [comment]
I feel like I should comment that this post is not https://github.com/elementary/edw/discussions/2 which is another thing 😅

**Reply by cassidyjames** [reply]
Ha good call, I edited the title for hopefully a little more clarity.

**Comment by cassidyjames** [comment]
We decided to go a different way with this so this is withdrawn.
